crates/iota-core/src/authority/shared_object_congestion_tracker.rs

```diff
@@ -67,6 +67,12 @@ impl SharedObjectCongestionTracker {
             .expect("There must be at least one object in shared_input_objects.")
     }
 
+    // NOTE: this function will be rewritten anyway in the new sequencer
+    // (see PR #5763), so we simple return the certificate's gas price here.
```

Comment:
```suggestion
    // (see PR #6490), so we simple return the certificate's gas price here.
```
Those changes will now be made in #6490